ql/src/java/org/apache/hadoop/hive/ql/ddl/table/AbstractAlterTableOperation.java

```diff
@@ -35,7 +35,6 @@
 import org.apache.hadoop.hive.ql.ddl.DDLOperationContext;
 import org.apache.hadoop.hive.ql.ddl.DDLUtils;
 import org.apache.hadoop.hive.ql.ddl.table.constraint.add.AlterTableAddConstraintOperation;
-import org.apache.hadoop.hive.ql.exec.repl.util.ReplUtils;
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.